javaparser-core/src/main/java/com/github/javaparser/utils/Utils.java

```diff
@@ -50,6 +50,13 @@
         return o;
     }
 
+    public static String assertNonEmpty(String id) {
+        if (id == null || id.isEmpty()) {
```

Comment:
You mean the parameter name? Well i couldn't come up with a proper name ... or you mean something else? I could rename it to string or something